content/SCALE/SCALE22.12.md

```diff
@@ -54,8 +54,168 @@ To upgrade an existing SCALE install, log in to your SCALE web interface and go
 SCALE is developed as an appliance that uses specific Linux packages with each release. Attempting to update SCALE with `apt` or methods other than the SCALE web interface can result in a nonfunctional system.
 {{< /hint >}}
 
-## 22.12-BETA.2 
+## 22.12-RC.1
+
+**November 15, 2022**
+
+TrueNAS SCALE 22.12-RC.1 has been released and includes many new features and improved functionaltiy. SCALE 22.12-RC.1 features include:
+
+* Adds FIPS-validated SSL module
+* Adds the R50M to the Enclosure screen
+* Adds USB passthrough support and allows users to specify USB vendor/product IDs in the UI
+* Adds increased functionality in the new Storage screens that include overprovisioning on zpool creation and the ability to see the full name for datasets with long names
+* Adds support for creating S3 buckets in Cloud Sync Backups
+* Updates Kubernetes to 1.25 and Samba to 4.17.0.rc5
+
+## 22.12-RC.1 Change Log
+
+### Epic
+
+* [NAS-110327](https://ixsystems.atlassian.net/browse/NAS-110327) WebUI Refactoring
+* [NAS-116607](https://ixsystems.atlassian.net/browse/NAS-116607) FIPS Validated SSL Module
+
+### New Feature
+
+* [NAS-109036](https://ixsystems.atlassian.net/browse/NAS-109036) Overlayfs support for docker on zfs
```

Comment:
```suggestion
* [NAS-109036](https://ixsystems.atlassian.net/browse/NAS-109036) OverlayFS support for docker on zfs
```

Reply:
On the list of issues I'm not supposed to fix the typos or strange spellings. I'm resolving this comment but not committing it.